swfiles/@spinw/spinwave.m

```diff
@@ -854,8 +854,11 @@
     else
         % All the matrix calculations are according to White's paper
         % R.M. White, et al., Physical Review 139, A450?A454 (1965)
-        
-        gham = mmat(gComm,ham);
+        if useMex
+            gham = sw_mtimesx(gComm, ham);
```

Comment:
I think I should also add a release note for this. Using the BiFeO3 test (but adjusting J1 to give a supercell of 6x6x6 and using 15 q-points) this change reduces the execution time from `14.7 +/- 0.5 s` to `5.1 +/- 0.4 s` and dramatically reduces the memory required (hence why I had to reduce the supercell dimensions and the number of q-points to profile without this change - using 20 qpts gave me an out-of memory error on IDAaaS with 80GB RAM). Note this speed-up is only significant for large magnetic cells (in this case 216 atoms), e.g. it did not speed up the FMchain example.

Reply:
Done in [c5e297c](https://github.com/SpinW/spinw/pull/124/commits/c5e297c6b7e9b910e3cf152d5cda5da70078968d)!